lte/gateway/python/magma/subscriberdb/tests/protocols/diameter/common.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+# pylint: disable=W0223
+
```

Comment:
maybe will use a more descriptive name for this file `MockTransport` would be fine.